how does Wasabi select which coins to send (#1351)
zkSNACKs/WasabiDoc: docs/FAQ/FAQ-UseWasabi.md

![Wasabi Wallet Send tab](/Send.png "Wasabi Wallet Send tab")
:::

:::details
### How does Wasabi select which coins to send?

> Wasabi automatically first selects the private coins. 

Author: ```suggestion
If the user has a coin with the same label, then Wasabi automatically first selects this coin. 
```

If there are no/not enough private coins for this transaction, then Wasabi will select private coins + non-private coins.

> Which non-private coins will be used can be selected by the user, based on the labelling system.

Author: why? the user selects at "who should know about this transaction" which label to use

![Wasabi Wallet Send tab](/Send.png "Wasabi Wallet Send tab")
:::

:::details
### How does Wasabi select which coins to send?

> If there is a coin with the same label (as the recipient label of the outgoing transaction), then Wasabi automatically first selects this coin.
> If this is not the case, or the labelled coin is not enough, then Wasabi will select private coins.

Reviewer: ```suggestion
If the user has coins with the same label (as the recipient label of the outgoing transaction), then Wasabi automatically selects these coins.
If there are no such coins, or they are not enough, then Wasabi will select private and semi private coins if necessary.
```

> If there are no/not enough private coins for this transaction, then Wasabi will select private coins + non-private coins.

Reviewer: ```suggestion
If there are no/not enough private and semi-private coins for this transaction, then Wasabi will select non-private coins also.
```

> Which non-private coins will be used can be selected by the user, based on the labelling system.

Reviewer: ```suggestion
The user can change which non-private coins will be used, based on the labelling system.
```
